far/FarRus.hlf.m4

```diff
@@ -5519,6 +5520,15 @@ Left/Right и Num4/Num6 действуют по-разному в зависим
 
     Изменение этого параметра возможно через ~far:config~@FarConfig@
 
+@Panel.Layout.DoubleGlobalColumnSeparator
+$ #far:config Panel.Layout.DoubleGlobalColumnSeparator#
```

Comment:
Same as in English file here and below - maybe we can reference existing topics?

Reply:
Ditto.

Reply:
Ditto again :)

Reply:
Sorry for missing this before. Too many minor changes; just lost track of the things. Sure, I will make it an alias (in both files). Thank you for carefully verifying the changes.